agr_literature_service/api/crud/reference_crud.py

```diff
@@ -409,3 +416,92 @@ def merge_references(db: Session,
     db.delete(old_ref)
     db.commit()
     return new_curie
+
+
+def get_citation_from_args(authorNames: str, year: str, title: str, journal: str,
+                           volume: str, issue: str, page_range: str):
+    # Create the citation from the args given.
+    citation = "{}, ({}) {} {} {} ({}): {}".\
+        format(authorNames, year, title,
+               journal, volume, issue, page_range)
+    return citation
+
+
+def author_order_sort(author: AuthorModel):
```

Comment:
This isn't being used ?

Reply:
Yep, I changed the way I did things in the end. Will Remove later.